app/src/controllers/log/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+import { fetchReducer } from 'controllers/fetch';
+import { CHANGE_ACTIVE_LOG_ITEM, SET_LOG_ITEM_TO_GET_HISTORY, NAMESPACE } from './constants';
+
+export const activeItemReducer = (state = '', { type, payload }) => {
+  switch (type) {
+    case CHANGE_ACTIVE_LOG_ITEM:
+      return payload;
+    default:
+      return state;
+  }
+};
+
+export const itemToGetHistoryReducer = (state = '', { type, payload }) => {
+  switch (type) {
+    case SET_LOG_ITEM_TO_GET_HISTORY:
+      return payload;
+    default:
+      return state;
+  }
+};
+
+export const logReducer = combineReducers({
+  logEntries: fetchReducer(NAMESPACE),
+  itemToGetHistory: itemToGetHistoryReducer,
```

Comment:
What is this for?
If it is an id, it supposed to be in an url.
If it is a whole item, it doesn't look like a proper name for it.
And what is the difference between this and `activeItemId`?

Reply:
This is the identifier of the item for which we want to load the history.
We request the history for this Id and hold it in state. But `activeItemId` - Id of item, which was selected in current history scope (scope for `itemToGetHistory`).
Understand naming problem.
`itemToGetHistory` must be set in the state when we follow the link to the itemId for which we want to load the history.

Reply:
If we have this parameter in the url, it's already in the store.

Reply:
Fixed